ChangeLog

```diff
@@ -26,6 +26,9 @@ Release date: Undefined
 
   Closes #4301
 
+* Fix crash when checking ``setup.cfg`` for pylint config when there are non-ascii characters in there
+
+  Closes #4328
```

Comment:
You probably need to add an empty line here (after `Closes ...`). If you like, you can do `pre-commit install` which will install git pre commit hooks that automatically check for such issues.

Reply:
Done.